scones/app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>House of Scones</title>
+  <%= csrf_meta_tags %>
+
+  <link href='https://fonts.googleapis.com/css?family=Josefin+Slab:400,700' rel='stylesheet' type='text/css'>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+</head>
+
+<body>
+  <h1>• Game of Scones •</h1>
+  <nav>
+    <a href="/houses">View Houses  •  </a>
+    <a href="/scones">View Scones  •  </a>
+    <a href="/houses/new">Create New House</a>
+  </nav>
+
+  <% flash.each do |type, message| %>
+  <p class="flash <%= type %>"><%= message %></p>
+  <% end %>
+
+  <%= yield %>
+</body>
+</html>
```

Comment:
seems like you really put your nose to the grindscone when you coded this layout 👍